### ISSUE TITLE
System.Data.SqlClient | UWP ARM Build

### ISSUE THREAD

**Issue by Kyagos** [dotnet/corefx#9064]
## Win 10 IoT ARM Build

The .NET Core Foundation Libraries already got Builds for most common platforms and architectures.

Since the release of Windows 10 IoT Preview Version and announced support for Raspberry Pi Products many developers want to use it in a commercial way but the current build does not inherit the full core. The main issue with the current IoT-Build is the missing ability to connect to MS SQL Server and pass your requests. Using an additional WCF Service and communicate over REST-API is not the proper way to transfer Data in a company. 

So my request is to add a build of the .NET CORE on ARM for Windows 10 IoT usage on the github list or at least adding the ability to handle SQL for MS SQL Server.

**tl;dr**
No possibility to connect to MS SQL Server via UWP 
Should be implemented since Raspberry Pi 3 got Win 10 IoT and is rising interest for companies

Please consider:
According to the porting documentation and a missing branch for IoT ARM I am leaving this request/issue on the master page.
There is no other project handling my request

**Comment by danmosemsft** [comment]
@saurabh500 is this scheduled for UWP6 (what we're currently working on)? If not please share update and fix milestone.

**Comment by saurabh500** [comment]
@danmosemsft is this the uapaot work that is underway?

**Comment by danmosemsft** [comment]
Yes. I think this is asking whether SqlClient will be available for UWP. I don't know know your plans or whether you have something to share right now. Moved to Future meantime.

**Comment by danmosemsft** [comment]
Linking this here https://wpdev.uservoice.com/forums/110705-universal-windows-platform/suggestions/9327945-uwp-sql-server-api

**Comment by divega** [comment]
As recently [announced in the .NET Blog](https://devblogs.microsoft.com/dotnet/introducing-the-new-microsoftdatasqlclient/), focus on new SqlClient features an improvements is moving to the new [Microsoft.Data.SqlClient package](https://www.nuget.org/packages/Microsoft.Data.SqlClient/). For this reason, we are moving this issue to the new repo at https://github.com/dotnet/SqlClient. We will still use https://github.com/dotnet/corefx to track issues on other providers like System.Data.Odbc and System.Data.OleDB, and general ADO.NET and .NET data access issues.
